README.md

```diff
@@ -86,107 +86,104 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    mkdir instructlab
    cd instructlab
    ```
 
-   > **NOTE:** The following steps in this document use [Python venv](https://docs.python.org/3/library/venv.html) for virtual environments. However, if you use another tool such as [pyenv](https://github.com/pyenv/pyenv) or [Conda Miniforge](https://github.com/conda-forge/miniforge) for managing Python environments on your machine continue to use that tool instead. Otherwise, you may have issues with packages that are installed but not found in your virtual environment.
-   <!-- -->
-   > **NOTE:** Some Python version management tools that build Python (instead of using a pre-built binary) may not by default build libraries implemented in C, and CPython when installing a Python version. This can therefore result in the following error when running the `ilab data generate` command: `ModuleNotFoundError: No module named '_lzma'`. This can be resolved by building CPython during the Python installation with the `--enable-framework`. For example for `pyenv` on MacOS: `PYTHON_CONFIGURE_OPTS="--enable-framework" pyenv install 3.x`. You may need to recreate you virtual environment after reinstalling Python.
+2. There are a few ways you can locally install the `ilab` CLI. Select your preferred installation method from the following instructions. You can then install `ilab` and activate your `venv` environment.
 
-3. There are a few ways you can locally install the `ilab` CLI. Select your preferred installation method from the following instructions. You can then install `ilab` and activate your `venv` environment.
+   ⚠️ The `python3` binary shown in the installation guides are the Python version that you installed in the above step. The command can also be `python3.11` or `python3.10` instead of `python3`. You can check Python's version by `python3 -V`.
 
-   > **NOTE**: ⏳ The `python3` binary shown in the following steps is the Python version that you installed in the above step. The command can also be `python3.11` or `python3.10` instead of `python3`. You can check Python's version by `python3 -V`.
-   >
-   > **NOTE**: ⏳ `pip install` may take some time, depending on your internet connection. In case installation fails with error ``unsupported instruction `vpdpbusd'``, prepend `CMAKE_ARGS="-DLLAMA_NATIVE=off"` to `pip install` command.
+   ⏳ `pip install` may take some time, depending on your internet connection. In case installation fails with error ``unsupported instruction `vpdpbusd'``, append `-C cmake.args="-DLLAMA_NATIVE=off"` to `pip install` command.
 
-   See [the GPU acceleration documentation](./docs/gpu-acceleration.md) for how to
-   to enable hardware acceleration for interaction and training on AMD ROCm,
-   Apple Metal Performance Shaders (MPS), and Nvidia CUDA.
+   See [the GPU acceleration documentation](./docs/gpu-acceleration.md) for how to to enable hardware acceleration for interaction and training on AMD ROCm, Apple Metal Performance Shaders (MPS), and Nvidia CUDA.
```

Comment:
This doc may need to be updated as well: https://github.com/instructlab/instructlab/issues/2237

Reply:
Also, it seems the below sections get into this anyway?

Reply:
I think it uses the hardware by default when the config is created. So I may just remove this reference until that doc is updated. 
WDYT?

Reply:
Tbh I think it'd be better to address the issue as part of this PR - either updating that document or moving the content here and removing it entirely.

Reply:
i honestly think we should unlink this doc, its super out of date and i have gotten dms regarding the info in it

Reply:
Not just unlink then, let's remove it outright. I just want to make sure no info from there is lost.

Reply:
So do yall think it would be best to just delete the gpu-acceleration.md file?

Reply:
if all the relevant info is copied elsewhere, yes

Reply:
I just went over the gpu-acceleration.md and most of the install parts were copied over to the README. So that content is covered. Im just not sure about some of additional cuda/rocm arguments that im unfamiliar with. Might need to sync with someone just for clarifications,